server/gameconnection.py

```diff
@@ -547,6 +571,19 @@ async def handle_game_full(self):
         """
         pass
 
+    async def handle_established_peers(self, peerId: str, peerConnectedTo: str):
+        """
+        Send by the lobby when a peer connected to another peer. Message can be send multiple times, by multiple connections.
+
+        # Params
+        - peerId: The identifier of the peer that this connection received the message from
```

Comment:
Add backticks so it matches the docstring for teamkill_happened. These get rendered as Markdown in our docs.
```suggestion
        - `peerId`: The identifier of the peer that this connection received the message from
```